ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -454,7 +471,7 @@ private[samplers] object BigSamplerBigQuery {
         val samplePct100 = samplePct * 100.0
         val r = sc.bigQueryTable(inputTbl)
           .flatMap { e =>
-            val hasher = BigSampler.kissHashFun(seed).newHasher(fields.size)
```

Comment:
What is the effect of removing the `newHasher`? As I understand it shouldn't matter much but should verify.

Reply:
I changed it to pass around the `Hasher` (essentially a builder for hashes) once it was already generated, rather than the `HashFunction` which initializes a hasher. This was necessary because we want to salt FarmHash with the "seed" param once it's already created, but we only know about the seed when we're initializing, not when we're adding existing fields (and we don't want to care about the "seed" as a murmur seed vs as farmhash salt when we're adding fields, because that would leak hash implementation details)